### PR TITLE
Add `describeContact` API method for Amazon Connect

### PR DESCRIPTION
**Pull request by elpete**

### CODE REVIEW

File: services/connect.cfc [hunk 1]
```diff
@@ -38,6 +38,27 @@ component {
         return apiResponse;
     }
 
+    /**
+    * Describes the specified contact.
+    * Contact information remains available in Amazon Connect for 24 months, and then it is deleted.
+    * Only data from November 12, 2021, and later is returned by this API.
+    * https://docs.aws.amazon.com/connect/latest/APIReference/API_DescribeContact.html
+    *
+    * @preview
```
Comment (jcberquist) [inline services/connect.cfc:47]: It looks like you still need to add parameter hints here?

**Comment by elpete** [comment]
Added docblocks

**Comment by jcberquist** [comment]
Thanks!

**Comment by elpete** [comment]
Thank you!

**Pull request merged**